llm_studio/src/metrics/text_causal_language_modeling_metrics.py

```diff
@@ -28,7 +28,11 @@ def sacrebleu_score(
     for predicted_text, target_text in zip(
         results["predicted_text"], results["target_text"]
     ):
-        scores.append(metric.sentence_score(predicted_text, [target_text]).score)
+        if target_text == "":
```

Comment:
you sure it's gonna be still "" after reading the pandas df or NaN ?

Reply:
This should cast None to str: `df[input_cols + [cfg.dataset.answer_column]].fillna("").astype(str)`

Reply:
yeah that should be fine then